modules/uncertainty/R/ensemble.R

```diff
@@ -464,7 +464,7 @@ for (input_tag in names(settings$run$inputs)) {
   
      if (!input_tag %in% names(samples)) {
         # Use first path (already validated as single path)
-        settings$run$inputs[[input_tag]]$path <- input$path[1]  } 
+        settings$run$inputs[[input_tag]]$path <- unlist(input$path[1])} 
```

Comment:
Just to confirm: This has been tested after #3561, correct? I think they should be compatible (That was just for the one-input case whereas this will also work on each member of a larger ensemble), but wanted to check.

Reply:
Yes. I tested using the newest version of pecan.